scripts/clean-linters.sh

```diff
@@ -0,0 +1,5 @@
+set -x
+set -e
+
+rm -rf pmd-bin*
```

Comment:
Maybe this is a bit messy to put a * behind everything... it would be nicer like this:
```
rm pmd-bin-*
rm -rf pmd-bin/
rm -rf tidy-html5/
```
Will change this.